tests/wallet/sync/test_wallet_sync.py

```diff
@@ -234,13 +236,13 @@ async def test_almost_recent(
         # Tests a reorg with the wallet
         ph = both_phs[i % 2]
         all_blocks = bt.get_consecutive_blocks(1, block_list_input=all_blocks, pool_reward_puzzle_hash=ph)
-        await full_node_api.full_node.add_block(all_blocks[-1])
+        await full_node.add_block(all_blocks[-1])
```

Comment:
@AmineKhaldi I'm not sure this is right. This test is meant to exercise `FullNodeApi`, isn't it? here it seems you're circumventing the code that's supposed to be tested

Reply:
maybe I'm wrong. I suppose this test is mostly concerned about what happens to the wallet

Reply:
@AmineKhaldi @arvidn was there a resolution to this thread?

Reply:
`full_node` is `full_node_api.full_node`, I think arvid noticed that in the second comment after sending the first.